### ISSUE TITLE
A question about the code

### ISSUE THREAD

**Issue by ZhangYiyangKevin** [Ontheway361/dul-pytorch#5]
Hi, it's really a nice job.
After reading this code, i have a question about the training stage. In line 118~120 of train.py file, if in training stage, the feat_mu represents the sample s(i). But when calculating the loss on line 120, the third parameter of criterion function needs the mu(i), not the sample s(i) named feat_mu. 
I wish i made myself clear. Looking forward to your reply, thanks.

**Comment by JieChang-M** [comment]
Yep, I also noticed this mistake. BTY, I am the author of this paper, you can e-mail me to get the code without dataset module.

**Comment by JieChang-M** [comment]
> Yep, I also noticed this mistake. BTY, I am the author of this paper, you can e-mail me to get the code without dataset module.

My e-mail:  changjie@megvii.com

**Comment by milliema** [comment]
@JieChang-M Hi! Tried to contact via email yet the above email address is no more valid, I'd like to check about the official source code, is it still available?

**Comment by Ontheway361** [comment]
您好，我已经收到您的邮件，谢谢。

**Comment by pswena** [comment]
> 您好，我已经收到您的邮件，谢谢。

你好，我刚刚给上面的邮箱发邮件了，但是邮件不存在，所以想请问你要一份代码，行吗？
